pkgs/tools/wayland/sway-audio-idle-inhibit/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, libpulseaudio
+, wayland
+, wayland-protocols
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
The `rec` doesn't do anything here